boards/arm/stm32f746g_disco/doc/stm32f746g_disco.rst

```diff
@@ -0,0 +1,199 @@
+.. _stm32f746g_disco_board:
+
+ST STM32F746G Discovery
+#######################
+
+Overview
+********
+
+The discovery kit enables a wide diversity of applications taking benefit
+from audio, multi-sensor support, graphics, security, security, video,
+and high-speed connectivity features. Important board features include:
+
+- STM32F746NGH6 microcontroller featuring 1 Mbytes of Flash memory and 340 Kbytes of RAM, in BGA216 package
+- On-board ST-LINK/V2-1 supporting USB re-enumeration capability
+- Five power supply options:
+
+    - ST LINK/V2-1
+    - USB FS connector
+    - USB HS connector
+    - VIN from Arduino connector
+    - External 5 V from connector
+
+- Two pushbuttons (user and reset)
+- USB functions: virtual COM port, mass storage, debug port
+- 4.3-inch 480x272 color LCD-TFT with capacitive touch screen
+- SAI audio codec
+- Audio line in and line out jack
+- Stereo speaker outputs
+- Two ST MEMS microphones
+- SPDIF RCA input connector
+- 128-Mbit Quad-SPI Flash memory
+- 128-Mbit SDRAM (64 Mbits accessible)
+- Connector for microSD card
+- USB OTG HS with Micro-AB connectors
+- USB OTG FS with Micro-AB connectors
+- Ethernet connector compliant with IEEE-802.3-2002
+
+.. image:: img/en.stm32f746g-disco.jpg
+     :width: 500px
+     :align: center
+     :height: 357px
+     :alt: STM32F746G-DISCO
+
+More information about the board can be found at the `32F746G-DISCO website`_.
+
+Hardware
+********
+
+The STM32F746G Discovery kit provides the following hardware components:
+
+- STM32F746NGH6 in BGA216 package
+- ARM |reg| 32-bit Cortex |reg| -M7 CPU with FPU
+- 216 MHz max CPU frequency
+- VDD from 1.8 V to 3.6 V
+- 2 MB Flash
+- 384+4 KB SRAM including 64-Kbyte of core coupled memory
+- GPIO with external interrupt capability
+- LCD parallel interface, 8080/6800 modes
+- LCD TFT controller supporting up to XGA resolution
+- MIPI |reg|  DSI host controller supporting up to 720p 30Hz resolution
+- 3x12-bit ADC with 24 channels
+- 2x12-bit D/A converters
+- RTC
+- Advanced-control Timer
+- General Purpose Timers (17)
+- Watchdog Timers (2)
+- USART/UART (8)
+- I2C (3)
+- SPI (6)
+- 1xSAI (serial audio interface)
+- SDIO
+- 2xCAN
+- USB 2.0 OTG FS with on-chip PHY
+- USB 2.0 OTG HS/FS with dedicated DMA, on-chip full-speed PHY and ULPI
+- 10/100 Ethernet MAC with dedicated DMA
+- 8- to 14-bit parallel camera
+- CRC calculation unit
+- True random number generator
+- DMA Controller
+
+More information about STM32F746NGH6 can be found here:
+
+       - `STM32F746NGH6 on www.st.com`_
```

Comment:
These two lines still need to be not intended...

Reply:
Shippable prints warnings like this:
"B4 Second line is not empty". Does this mean that commit message body should have only one line?
Also there is one failed test, but I don't understand how check which exactly test was failed. Are these tests from sanity check?